### PR TITLE
Fix for "Could not get event record" error and cleaned up logging

### PR DESCRIPTION
**Pull request by sana-jawad**
1. Fix for error seen while ingesting data

Processing for updateEventCountTable failed with error Could not get event record: invalid key for table eventcount: /eventcount/001630076400/Pod/namespace/podname: key should have 6 parts: 

2. Also cleaned up logging and added a configuration to change the setting as needed.
3. Update debug page to show data from all tables.

### CODE REVIEW

File: pkg/sloop/processing/watch.go [hunk 1]
```diff
@@ -124,3 +126,21 @@ func toWatchTableKeyPrefix(ts *timestamp.Timestamp, kind string, namespace strin
 
 	return typed.NewWatchTableKey(untyped.GetPartitionId(timestamp), kind, namespace, name, time.Time{}), nil
 }
+
+func GetUidForWatchEntry(tables typed.Tables, txn badgerwrap.Txn, kind string, namespace string, name string, timestamp time.Time) (string, error) {
+	watchKeyComparator := typed.NewWatchTableKeyComparator(kind, namespace, name, timestamp)
+	seekKey := queries.GetSeekKey(watchKeyComparator, time.Now())
```
Comment (akrag) [inline pkg/sloop/processing/watch.go:132]: Do we need a nil check here?

File: pkg/sloop/processing/eventcount.go [hunk 2]
```diff
@@ -69,6 +70,15 @@ func updateEventCountTable(
 
 	eventCountByMinute := spreadOutEvents(computedFirstTs, computedLastTs, computedCount)
 
+	if involvedObject.Uid == "" {
```
Comment (akrag) [inline pkg/sloop/processing/eventcount.go:73]: can involvedObject be nil here? Are we receiving only involvedObject without Uid here?

File: pkg/sloop/processing/eventcount.go [hunk 3]
```diff
@@ -69,6 +70,15 @@ func updateEventCountTable(
...
+		glog.V(common.GlogVerbose).Infof("Got empty Uid for name: %v, namespace: %v, kind: %v for event: %v ", involvedObject.Name, involvedObject.Namespace, involvedObject.Kind, newEventInfo.Reason)
+		returnedUid, err := GetUidForWatchEntry(tables, txn, involvedObject.Kind, involvedObject.Namespace, involvedObject.Name, time.Time{})
+		if err != nil {
+			return err
```
Comment (akrag) [inline pkg/sloop/processing/eventcount.go:77]: errors.Wrap(err, "Error retrieving Uid for the Watch Entry") ?

File: pkg/sloop/processing/eventcount.go (hunk 2 above)
Comment (sana-jawad) [reply pkg/sloop/processing/eventcount.go:73]: No Its already checked here https://github.com/salesforce/sloop/blob/master/pkg/sloop/processing/processing.go#L56 Yes we are receiving some involved objects with Uid empty

File: pkg/sloop/processing/watch.go (hunk 1 above)
Comment (sana-jawad) [reply pkg/sloop/processing/watch.go:132]: All the fields have already been checked in parent function

File: pkg/sloop/processing/eventcount.go (hunk 3 above)
Comment (sana-jawad) [reply pkg/sloop/processing/eventcount.go:77]: sure! updated

**Pull request closed without merging**